$OpenBSD: patch-src-make_rules,v 1.7 2010/09/29 18:45:49 jasper Exp $

XXX make it compile with gtk+2 2.22 (*_DISABLE_DEPRECATED) and separate
gdk-pixbuf:
./gtk/file/gamgi_gtk_file_chooser.c:429: error: expected declaration specifiers or '...' before 'GtkNotebookPage

--- src/make_rules.orig	Sun Jul  4 20:56:55 2010
+++ src/make_rules	Wed Sep 29 20:20:48 2010
@@ -18,7 +18,8 @@ CC ?= gcc
 
 CFLAGS ?= -O3 -Wall -ansi
 override CFLAGS += -DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED \
-	-DGDK_PIXBUF_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED \
+	-DGDK_PIXBUF_DISABLE_DEPRECATED \
+	`pkg-config --cflags gdk-pixbuf-2.0` -pthread
 #	-DGSEAL_ENABLE \
 #	-DG_DISABLE_SINGLE_INCLUDES -DATK_DISABLE_SINGLE_INCLUDES \
 #	-DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES
@@ -37,7 +38,7 @@ EXE = gamgi
 LIBS = -lgtkglext-x11-1.0 -lgdkglext-x11-1.0 -lGLU -lGL -lgtk-x11-2.0 \
 	-lgdk-x11-2.0 -latk-1.0 -lgdk_pixbuf-2.0 -lpangocairo-1.0 \
 	-lpango-1.0 -lcairo -lgobject-2.0 -lgmodule-2.0 -lglib-2.0 \
-	-lXext -lX11 -lXi -lXmu -lfreetype -lexpat -lm -ldl
+	-lXext -lX11 -lXi -lXmu -lfreetype -lexpat -lm -pthread
 
 #================================ full paths ===============================
 
